tests/Common/ComponentsTest.php

```diff
@@ -245,10 +232,11 @@ public function testComponentConfigRenew()
     }
 
     /**
-     * @dataProvider provideComponentsClientType
-     * @param string $clientType
+     * on defaultBranch only, no need devBranch because "Deleting configuration from trash is not allowed in
+     * development branches."
+     * @return void
      */
-    public function testComponentConfigDelete($clientType)
```

Comment:
Tady ten dataProvider muze zustat, pac staci jen vynechat posledni call na purge. Viz podminka na radku 304 (https://github.com/keboola/storage-api-php-client/pull/722/files?diff=split&w=1#diff-2726ab8159393b2a45bafacf5d6e3b421b0de65fd569e2f0d96339a7e0b83c8aL334)

Reply:
Respektive Tomasova uprava: https://github.com/keboola/storage-api-php-client/pull/706/commits/3f849fc7cd67a37adb4058d3ca3d0ac970a129f6

Reply:
Takze je tam error: `Undefined variable '$clientType' `

Reply:
Jak na to koukam, to by slo sjednotit do jednoho s podminkama pro
- `purge` v `default` branchi
- `// check main branch` v `dev` branchi

Je to dobrej napad?

Reply:
dohodli jsme se
- mrknu jestli testComponentConfigDelete a BranchCompoenntTest::testDeleteBranchConfiguration dela cca to same
- pokud ano, tak tady to pustime bez providera a smazu podminku na konci
- zkusim tady ten test trochu okomentovat jako to ma ten test v branchi at to je trochu prehledne